docs/types/field.md

```diff
@@ -1,19 +1,22 @@
 ## Fields
 
-_Fields_ are the primary way to tell stanza which values of an entry to use in its operators.
-Most often, these will be things like fields to parse for a parser operator, or the field to write a new value to.
+A _Field_ is a reference to a value in a log [entry](/docs/types/field.md). 
 
-Fields are `.`-delimited strings which allow you to select labels or records on the entry. Fields can currently be used to select labels, values on a record, or resource values. To select a label, prefix your field with `$label` such as with `$label.my_label`. For values on the record, use the prefix `$record` such as `$record.my_value`. For resource values, use the prefix `$resource`.
+Many [operators](/docs/operators/README.md) use fields in their configurations. For example, parsers use fields to specify which value to parse and where to write a new value.
 
-If a key contains a dot in it, a field can alternatively use bracket syntax for traversing through a map. For example, to select the key `k8s.cluster.name` on the entry's record, you can use the field `$record["k8s.cluster.name"]`.
+Fields are `.`-delimited strings which allow you to select labels or records on the entry. 
```

Comment:
Are "labels" the same things as "attributes"? If it is then it will great to change this to be consistent with Otel vocabulary.

Reply:
Yes, this is a good point. We can definitely do this. 

Tracking issue: https://github.com/open-telemetry/opentelemetry-log-collection/issues/22